package.json

```diff
@@ -9,30 +9,34 @@
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
     "build": "yarn build-ts && yarn lint",
     "dev:watch": "nodemon -r ts-node src/index.ts",
-    "test": "NODE_ENV=test jest --coverage",
+    "test": "jest --coverage",
+    "pretest": "NODE_ENV=test yarn db:migrate:undo && NODE_ENV=test yarn db:migrate && NODE_ENV=test yarn db:seed:undo && NODE_ENV=test yarn db:seed",
```

Comment:
What happens when you only write a single NODE_ENV=test from the beginning of the script? Instead of repeating it before each command.